db/models.py

```diff
@@ -1 +1,43 @@
 from django.db import models
+
+
+class Race(models.Model):
+    race_to_choose = (
+        ("Elf", "Elf"),
+        ("Dwarf", "Dwarf"),
+        ("Human", "Human"),
+        ("Ork", "Ork")
+    )
+    name = models.CharField(
+        max_length=255,
+        unique=True,
+        choices=race_to_choose
+    )
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True
+    )
+    bonus = models.CharField(
+        max_length=255
+    )
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
```

Comment:
Consider adding `blank=True` to the `description` field in the `Guild` model to allow it to be optional in forms, similar to the `description` field in the `Race` model.